### PR TITLE
changing np to cnp

### LINKED ISSUE #3908: change `cimport numpy as np` to `cimport numpy as cnp` for consistency
## Is your feature request related to a problem? ##

We should be consistent with the way we cimport numpy in the codebase. 

Lots of places currently use
```cython
 cimport numpy as np
```

While the more modern approach and the one taken in some PRs #3777 , `timestep.pyx` and a few random other existing places is to

```cython 
 cimport numpy as cnp
```

Personally I prefer the second one (and its the one used by pandas) as it is much clearer when the C API is being used vs python API. 
@tylerjereddy  do you know if there is a standard for scipy?

Regardless of which way we go we should be consistent if possible.

Let me know what you think on this issue and if we come to a conclusion I can raise a seperate issue for taking some action. 
Might be a nice easy `good-first-issue` etc.

### PR DESCRIPTION
**Pull request by HunterZ17**
Fixes #3908 

Changes made in this Pull Request:
 - changed `cimport numpy as np` to `cimport numpy as cnp` and it's related uses of `np` to `cnp`

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

----
:books: Documentation preview :books:: https://readthedocs-preview--4101.org.readthedocs.build/en/4101/

### CODE REVIEW

**Review by richardjgowers — CHANGES_REQUESTED** [review]
@HunterZ17 thanks for starting this.  It looks like there's some test failures from where you can't use the `cnp` Cython module but instead should use the `np` (Python) import

**Comment by HunterZ17** [comment]
@richardjgowers So, Should I run tests locally and replace the needed `cnp` with `np`.

**Comment by RMeli** [comment]
@HunterZ17 Yes, you should understand which imports should be `np` and which one should be `cnp`. Testing thing locally will allow you to iterate much more quickly and push to this PR when things are working.

**Comment by richardjgowers** [comment]
Yes that should make it work

File: package/MDAnalysis/lib/_augment.pyx [hunk 1]
```diff
@@ -296,12 +296,12 @@ def augment_coordinates(float[:, ::1] coordinates, float[:] box, float r):
                 output.push_back(coord[j] - shiftZ[j])
             indices.push_back(i)
     n = indices.size()
-    return np.asarray(output, dtype=np.float32).reshape(n, 3), np.asarray(indices, dtype=np.intp)
+    return np.asarray(output, dtype=cnp.float32).reshape(n, 3), np.asarray(indices, dtype=cnp.intp)
```
Comment (Telomelonia) [inline package/MDAnalysis/lib/_augment.pyx]: ig `float32` and `intp` are of numpy module rather than `cnp`

**Comment by HunterZ17** [comment]
After testing locally, all `.pyx` files were running smoothly but all `.pxd` files were showing errors, always some methods are not defined as errors. Is there a different method to test `.pxd` file?

**Comment by hmacdope** [comment]
@HunterZ17  do you plan on continuing to work on this? If not we will close.

**Comment by HunterZ17** [comment]
@hmacdope Yes, you can close this.

**Pull request closed without merging**